### PR TITLE
GIBCT profile fixes

### LINKED ISSUE #5404: GI Bill CT: Hide drawers & info for OJT programs
This can be backlogged.

The following drawers and info should be hidden for OJT programs:
* Veterans Programs (they are not eligible)
* Student Outcomes (not applicable)
* Institutional Summary (in additional information drawer)

image

@leannammiller @saneshark @mphprogrammer

### PR DESCRIPTION
**Pull request by U-DON**
Resolves department-of-veterans-affairs/vets.gov-team#2444.
Resolves department-of-veterans-affairs/vets-website#5404.

Also fixed some excessive spacing issues on certain lines.

### CODE REVIEW

File: src/js/gi/components/profile/AdditionalInformation.jsx [hunk 1]
```diff
@@ -30,9 +32,48 @@ export class AdditionalInformation extends React.Component {
...
+          <strong>
+            <a onClick={this.props.onShowModal.bind(this, 'creditTraining')}>
+              Credit for military training:
+            </a>
+          </strong>
+          &nbsp;{!!it.creditForMilTraining ? 'Yes' : 'No'}
+        </div>
+      </div>
+    );
+  }
```
Comment (U-DON) [inline src/js/gi/components/profile/AdditionalInformation.jsx]: Just moved the deleted block below into its own function so it's clearer to see the condition in which it's rendered.

**Review by melwoodard — APPROVED** [review]
Looks good!

**Comment by U-DON** [comment]
@patrickvinograd: Pinging for dev review.

**Review by patrickvinograd — APPROVED** [review]
LGTM!

**Pull request merged**